src/libs/defaultRedirection/defaultRedirection.ts

```diff
@@ -0,0 +1,131 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+import CozyClient, { deconstructCozyWebLinkWithSlug } from 'cozy-client'
+
+import { formatRedirectLink } from '/libs/functions/formatRedirectLink'
+import { logger } from '/libs/functions/logger'
+import { getErrorMessage } from '/libs/functions/getErrorMessage'
+import strings from '/constants/strings.json'
+
+export const log = logger('AppBundle')
+
+export const NAVIGATION_APP_SLUG = 'home'
+
+export const DEFAULT_REDIRECTION_DELAY_IN_MS = 3000
+
+export interface SettingsCollection {
+  get: (path: string) => Promise<InstanceSettings>
+}
+
+interface InstanceSettings {
+  data: {
+    attributes: {
+      default_redirection: string
+    }
+  }
+}
+
+export const isDefaultRedirectionUrlNavigationApp = (
+  defaultRedirectionUrl: string,
+  client: CozyClient
```

Comment:
Does the CozyClient import at the beginning of the file mandatory to be able to type this arg?

Reply:
Yes it is, but my understanding is that `tsc` would remove that import at build

Reply:
It seems so. I will ask confirmation.

Reply:
Update: `import type CozyClient from 'cozy-client'`